### DISCUSSION TITLE
内存查看与修改

### DISCUSSION THREAD

**Discussion by bczhc** [Physics365/991CN-X-CW-Decompilation#37] [General]
我想这样的东西应该对学习卡西欧exploits很有帮助。请问有比较完善的这样的项目吗？

初学，我刚在项目user202729/CasioEmu上搞了个这么个玩意，看起来确实有点意思！

视频：

[a.webm](https://github.com/Physics365/991CN-X-CW-Decompilation/assets/49330580/92d73046-ee17-4980-99bd-6d87c6c78b63)

它的一套lua都是提供了接口的，但交互起来还是感觉不太方便。我就直接就修改了程序，每0.1s转储出来内存，然后再用xxd显示出来。

BTW，我发现D318 D319这两个位置，是个小端的计数器。

**Comment by fish4terrisa-MSDSM** [comment]
> 我想这样的东西应该对学习卡西欧exploits很有帮助。请问有比较完善的这样的项目吗？
> 
> 初学，我刚在项目user202729/CasioEmu上搞了个这么个玩意，看起来确实有点意思！
> 
> 视频：
> 
>  a.webm 
> 它的一套lua都是提供了接口的，但交互起来还是感觉不太方便。我就直接就修改了程序，每0.1s转储出来内存，然后再用xxd显示出来。
> 
> BTW，我发现D318 D319这两个位置，是个小端的计数器。

只有我这边有问题吗。。。。什么都看不到的说

**Reply by Wenti-D** [reply]
可能只有你（

**Reply by fish4terrisa-MSDSM** [reply]
好像是手机端的bug的说

**Comment by Wenti-D** [comment]
[qiufuyu 的 CasioEmuNeo](https://github.com/qiufuyu123/CasioEmuNeo) 好像有内存查看器和编辑器来着？
![image](https://github.com/Physics365/991CN-X-CW-Decompilation/assets/73067578/3ad43239-2aad-44c7-aaca-55a74aadb2d4)

**Comment by bczhc** [comment]
> 只有我这边有问题吗。。。。什么都看不到的说

视频是av1的，我的firefox和chromium都是正常的。

> [qiufuyu 的 CasioEmuNeo](https://github.com/qiufuyu123/CasioEmuNeo) 好像有内存查看器和编辑器来着？

欧，感谢！我看看。
